loadtest/store/memstore/store.go

```diff
@@ -60,6 +60,11 @@ func (s *MemStore) SetUser(user *model.User) error {
 	if user == nil {
 		return errors.New("user should not be nil")
 	}
+
+	if s.user != nil && user.Password == "" {
```

Comment:
How about ignoring other stuff too, like AuthData and MfaSecret ? https://github.com/mattermost/mattermost-server/blob/fdd1ae974fc1c27a85378f9ed535119ab87331ec/model/user.go#L495

Reply:
I don't believe we'll be making use of that data any time soon but I guess it wouldn't hurt to ignore it so we don't get into this problem again. I'll update.